build_web_compilers/build.yaml

```diff
@@ -138,7 +138,7 @@ builders:
         dart2js_args:
           - --enable-asserts
       release_options:
-        compiler: dart2js
+        compiler: both
```

Comment:
I think we want something more specific for this name, which might end up being pretty verbose, but it is what it is. Possibly, `dart2wasm+dart2js` or something.